ts/src/coinbase.ts

```diff
@@ -3215,24 +3214,32 @@ export default class coinbase extends Exchange {
         //      ]
         //    }
         //
+        // or
+        // {
+        //      "error": "PERMISSION_DENIED",
+        //      "error_details": "Orderbook is in limit only mode",
+        //      "message": "Orderbook is in limit only mode"
+        // }
         let errorCode = this.safeString (response, 'error');
         if (errorCode !== undefined) {
-            const errorMessage = this.safeString (response, 'error_description');
-            this.throwExactlyMatchedException (this.exceptions['exact'], errorCode, feedback);
-            this.throwBroadlyMatchedException (this.exceptions['broad'], errorMessage, feedback);
-            throw new ExchangeError (feedback);
+            const errorMessage = this.safeString2 (response, 'error_description', 'message');
+            const message = errorCode + ': ' + errorMessage;
```

Comment:
I don't like the fact that you change `errorMessage` and `message` meaning from file to file: sometimes `errorMessage` is the original error message, sometimes you put the code with the message inside. Please, adapt one convention across all files you modified.

Reply:
right